contracts/AnchoredView/SymbolConfiguration.sol

```diff
@@ -0,0 +1,119 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+// verbose configuration of anchored view, transforming
+// address to symbols and such
+contract SymbolConfiguration {
```

Comment:
I do want to be weary about hiding complexity here. Is the advantage we're getting here simply that the code lives in a separate file?

Reply:
yeah, it was just a distraction for navigating the file. Happy to inline it later when not so much development happening